<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joinDialog</class>
 <widget class="QDialog" name="joinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>wechat-fake</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/wechat.png</normaloff>:/image/wechat.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/wechat.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="portWidget" native="true">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>259</y>
     <width>234</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="portLayout">
      <item>
       <widget class="QLabel" name="portlabel">
        <property name="text">
         <string>可用端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="portBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="portErr">
      <property name="styleSheet">
       <string notr="true">color=rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="nameWidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>234</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="nameLayout">
      <item>
       <widget class="QLabel" name="namelabel">
        <property name="text">
         <string>登录名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameEdit">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="nameErr">
      <property name="styleSheet">
       <string notr="true">color=rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
